config/nonprod.tfvars

```diff
@@ -0,0 +1,13 @@
+env = "nonprod"
+subusers = [
```

Comment:
I would just call this accounts as it's probably clearer
```suggestion
accounts = [
```

Reply:
done